### DISCUSSION TITLE
Move new tab `+` button back to bottom

### DISCUSSION THREAD

**Discussion by KaKi87** [zen-browser/desktop#1427] [Ideas]
Hi,
I would like to see the new tab `+` button back at the bottom please, as it makes sense for it to be there since new tabs appear at the bottom.
Thanks

**Comment by JomnTAL** [comment]
I agree. A toggle might be the way to go if some people prefer it at the top.

**Comment by DareDevil2000** [comment]
Strongly agreeing to this point ✊

**Comment by keiothic** [comment]
wait... the bottom most?? or the bottom of the last tab?? because I would love it to be just after the last tab where it makes much more sense

**Reply by KaKi87** [reply]
> wait... the bottom most??

No.

> or the bottom of the last tab??

Yes.

> because I would love it to be just after the last tab where it makes much more sense

Exactly what I said.
